netbox/resource_netbox_manufacturer_test.go

```diff
@@ -0,0 +1,72 @@
+package netbox
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"testing"
+
+	"github.com/fbreckle/go-netbox/netbox/client"
+	"github.com/fbreckle/go-netbox/netbox/client/dcim"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccNetboxManufacturer_basic(t *testing.T) {
+
+	testSlug := "device_type"
```

Comment:
c&p: this should be "manufacturer" but note that there are some length limits in netbox. Maybe use "mnfctrr" or "manuf" or sth

Reply:
I made it "manufacturer" for clarity. I see other tests have even longer slugs, such as "platform_basic".

Reply:
ah ok, as long as we need no suffix, thats okay, too. I just remember some things being too long, probably when I also used a long suffix for the slug.